consul/install.sls

```diff
@@ -0,0 +1,48 @@
+{% from "consul/map.jinja" import consul, consul_service with context %}
+
+# At the time of writing, Hashicorp doesn't maintain any official package
+# repositories for common linux distributions. The recommended installation
+# method is to download a zip, verify it, and extract consul into the system
+# path.
+
+include:
+  - .service
+
+download_package:
```

Comment:
This id is pretty generic. Maybe make it explicit that it is installing the Consul binary? That also makes it less likely to have a naming conflict if there are other states being included in a highstate run for a given minion.